classes/announcement/PKPAnnouncementDAO.inc.php

```diff
@@ -338,7 +338,7 @@ function &getAnnouncementsNotExpiredByAssocId($assocType, $assocId, $rangeInfo =
 			FROM announcements
 			WHERE assoc_type = ?
 				AND assoc_id = ?
-				AND (date_expire IS NULL OR date_expire > CURRENT_DATE)
+				AND (date_expire IS NULL OR date_expire >= CURRENT_DATE)
 				AND (date_posted < CURRENT_DATE)
```

Comment:
I think that the fix should go here, date_posted <= CURRENT_DATE, right? And thanks for your bug report and the pull request :)